pymc3/stats.py

```diff
@@ -342,7 +342,7 @@ def bpic(trace, model=None):
 
 
 def compare(traces, models, ic='WAIC', method='stacking', b_samples=1000,
-            alpha=1, seed=None):
+            alpha=1, seed=None, round_to=2):
```

Comment:
I still find it a bit too arbitrary. For example, with the default we are also rounding the `se` etc to 2 decimal point - do you still see the differences after rounding?